config.template.yml

```
@@ -553,4 +553,62 @@ notifier:
   #   sender: admin@example.com
   #   host: smtp.gmail.com
   #   port: 587
+
+##
+## Identity Providers
+##
+identity_providers:
+
+  ##
+  ## OpenID Connect (Identity Provider)
+  ##
+  ## It's recommended you read the documentation before configuraiton of this section:
+  ## https://www.authelia.com/docs/configuration/identity-providers/oidc.html
+  oidc:
+    ## The hmac_secret is used to sign the JWT's sent by OpenID Connect.
+    ## HMAC Secret can also be set using a secret: https://www.authelia.com/docs/configuration/secrets.html
+    hmac_secret: this_is_a_secret_abc123abc123abc
+
+    ## The issuer_private_key is used to encrypt the JWT's sent by OpenID Connect.
+    ## Issuer Private Key can also be set using a secret: https://docs.authelia.com/configuration/secrets.html
+    issuer_private_key: |
+      --- KEY START
+      --- KEY END
+
+    ## Clients is a list of known clients and their configuration.
+    clients:
+      -
+        ## The ID is the OpenID Connect ClientID which is used to link an application to a configuration.
+        id: myapp
+
+        ## The description to show to users when they end up on the consent screen. Defaults to the ID above.
+        description: My Application
+
+        ## The secret is a shared secret between Authelia and the consumer of this client.
+        secret: this_is_a_secret
+
+        ## The policy to require for this client; one_factor or two_factor.
+        policy: two_factor
```

```suggestion
authorization_policy: two_factor```

What do you think about being more explicit here?

---

Agree!